src/app/app.module.ts

```diff
@@ -39,10 +39,8 @@ export function markedOptions(): MarkedOptions {
 
   return {
     gfm: true,
-    tables: true,
     breaks: false,
     pedantic: false,
-    sanitize: false,
     smartLists: true,
     smartypants: false,
   };
```

Comment:
I think this is API changes in the object where tables and sanitize weren't in MarkedOptions anymore.